cypress/support/pages/contactForm.pageObject.js

```diff
@@ -3,24 +3,28 @@ import PageObject from '../PageObject';
 class ContactFormPageObject extends PageObject {
   url = '/index.html';
 
-  get emailField() {
-    return cy.get('#recipient-email');
+  get nameField() {
+    return cy.get('#name');
   }
 
-  get nameField() {
-    return cy.get('#recipient-name');
+  get countryField() {
+    return cy.get('#country');
+  }
+
+  get cityField() {
+    return cy.get('#city');
   }
 
-  get messageField() {
-    return cy.get('#message-text');
+  get cardField() {
+    return cy.get('#card');
   }
 
-  get sendMessageBtn() {
-    return cy.contains('.btn', 'Send message');
+  get monthField() {
+    return cy.get('#month');
   }
 
-  typeEmail(email) {
-    this.emailField.type(email, { force: true });
+  get yearField() {
+    return cy.get('#year');
   }
 
   typeName(name) {
```

Comment:
The function 'typeName' is missing the super call to the 'type' method. All inputs should be passed to a super method to handle typing into the field.